src/pages/index.tsx

```diff
@@ -138,8 +193,48 @@ const IndexPage: NextPage = () => {
         <ClientContent />
       ) : (
         <>
-          <Text marginBottom={4} as={"p"}>Install this app in your Dashboard and check extra powers!</Text>
-          {mounted && !global.location.href.includes("localhost") && <AddToSaleorForm />}
+          {isLocalHost && (
+            <Box>
+              <Text variant={"heading"} marginTop={8} as={"h2"}>
+                Installation
+              </Text>
+              <Text as={"p"} variant="body" marginTop={2}>
+                To install the app in your Saleor Dashboard, follow these steps:
+              </Text>
+              <ol>
+                <li>
+                  1. Copy the URL of your{" "}
+                  <a
+                    href="https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-with-tunnels"
```

Comment:
we have now 2 ways of doing it. Maybe lets give here 2 "paths" - with tunnels and locally. Just link two article to chose from

Reply:
Im also not sure if these steps are required.

The form below is basically doing the same thing but automatically, under the hood. 

IMO its just complicating things.

Learn about tunneling or local development - needed
Explain steps on dashboard - not needed